docs/backend/schema.ddl

```diff
@@ -0,0 +1,115 @@
+///////////////////////////////////////////////////
+// DPHs and users
+
+Table Dph {
+  id int [pk, increment]
+  name varchar [not null, unique]
+  gisaid_division_name varchar [unique]
+}
+
+Table User {
+  id int [pk, increment]
+  name varchar [not null]
+  auth0_user_id varchar [not null]
+  dph int [not null, ref: > Dph.id]
+  dph_admin bool [not null]
+}
+
+// members of dph can see data uploaded by owner_dph
+Table DphCanSee {
+  dph int [not null, unique, ref: > Dph.id]
+  owner_dph int [not null, unique, ref: > Dph.id]
+}
+
+///////////////////////////////////////////////////
+// Physical Samples
+
+Table PhysicalSample {
+  id int [pk, increment]
+  submitting_dph int [not null, ref: > Dph.id]
+  date_taken datetime [not null]
+  external_accession varchar [not null]
+  location varchar [not null]
+  extraction_method varchar [not null]
+  sample_source varchar [not null]
+  
+  // TODO: the other fields from PHA4GE
+  
+  Indexes {
+    (submitting_dph, external_accession) [unique]
+  }
+}
+
+Table Gene {
+  id int [pk, increment]
+  name varchar [not null, unique]
+}
+
+Table PcrPrimer {
+  id int [pk, increment]
+  gene_id int [not null, ref: > Gene.id]
+  name varchar [not null]
+  
+  Indexes {
+    (gene_id, name) [unique]
+  }
+}
+
+Table PcrCtValue {
+  physical_sample_id int [not null, ref: > PhysicalSample.id]
+  pcr_primer_id int [not null, ref: > PcrPrimer.id]
+  value float [not null]
+  
+  Indexes {
+    (physical_sample_id, pcr_primer_id) [unique]
+  }
+}
```

Comment:
@jackkamm would love your feedback on if this is adequate to capture CT values but is flexible enough for changes in the PCR assay.

Reply:
Honestly I'm a little hesitant to store all these CT values. There are so many assays, they are so heterogeneous, some of them don't even give CT values but give other readouts. While they have been generally useful for us in sanity checking our sequencing runs, for covidtracker v2 I think we will devolve this responsibility to the submitting DPHs

Reply:
@sidneymbell if you're on board with this, i can drop this.